src/main/resources/liquibase.xml

```diff
@@ -700,4 +700,36 @@
                    schemaName="public"
                    tableName="health_info_notification"/>
     </changeSet>
+    <changeSet id="202009291330" author="Neeraj" context="POC">
+        <preConditions>
+            <not>
+                <tableExists tableName="subscription_request"/>
+            </not>
+        </preConditions>
+        <comment>Add subscription_request table</comment>
+
+        <createTable tableName="subscription_request">
```

Comment:
when the user approves (and say in the process edits the subscription details), do you plan to save it in the same table? If so, then probably change the name of the table to just "hiu_subscription" .. with the status capable of storing values  like "REQUESTED", "GRANTED", "DENIED", "REVOKED" etc.